csrc/device_lower/lower2device.cpp

```diff
@@ -417,21 +417,11 @@ void GpuLower::analysis(Fusion* fusion) {
   // names
   if (this->requiresIdModel() || isOptionEnabled(EnableOption::IdModel)) {
     // Enable validation in the DEBUG build mode
-#ifdef NDEBUG
-    // Not DEBUG build
     id_model_ = std::make_unique<IdModel>(
         fusion_,
         /*build_graphs=*/true,
         /*allow_self_mapping=*/false,
         /*validate=*/false);
-#else
```

Comment:
Disabled the validation with ComputeAtMap since we now deal with non-conventional graph structures, so `ComputeAtMap` does not always work.